### ISSUE TITLE
The

### ISSUE THREAD

**Issue by Fantastic1115** [mila-iqia/cableinspect-ad-code#19]
I am a beginner and I am very interested in your research. 
Why does the test set in the label file I generated contain significantly more images than the training set?
